### PR TITLE
virtio-fs: new case of share data between vms

### PR DESCRIPTION
**Pull request by xiagao**
Support sharing data between windows vms.
ID: 1889560
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
(1/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.virtio_fs_multi_vms.share_fs_source: PASS (181.36 s)
 (2/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_multi_vms.share_fs_source: PASS (642.41 s)

**Comment by xiagao** [comment]
@YongxueHong
Call for your review, thank you..

File: qemu/tests/virtio_fs_multi_vms.py [hunk 1]
```diff
@@ -23,10 +26,51 @@ def run(test, params, env):
...
+        return exe_path
+
     cmd_dd = params.get('cmd_dd')
     cmd_md5 = params.get('cmd_md5')
     io_timeout = params.get_numeric('io_timeout')
     shared_fs_source_dir = params.get('shared_fs_source_dir')
+    os_type = params.get('os_type')
+
+    # cfg for windows vm
+    cmd_timeout = int(params.get("cmd_timeout", 120))
```
Comment (YongxueHong) [inline qemu/tests/virtio_fs_multi_vms.py:70]:
```suggestion
    cmd_timeout = params.get_numeric("cmd_timeout", 120)
```

Comment (xiagao) [reply qemu/tests/virtio_fs_multi_vms.py]: Updated @YongxueHong

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**